modules/fedimint-ln-client/src/lib.rs

```diff
@@ -903,7 +904,8 @@ impl LightningClientModule {
             .get_value(&MetaOverridesKey {})
             .await
         {
-            if meta.fetched_at.elapsed().unwrap() < META_OVERRIDE_CACHE_DURATION {
+            let elapsed = now().duration_since(meta.fetched_at).unwrap();
```

Comment:
Let's remove the `unwrap` here too and replace with `expect`

Reply:
done